lib/manager/tool-versions/__fixtures__/.tool-versions-with-ignore-comments

```diff
@@ -0,0 +1,2 @@
+ruby 2.5.3 # This is a comment
+nodejs 10.15.0 # renovate:ignore
```

Comment:
I'd like to see those small files inlined. We do it on most other manager too. That ways the test case is easier to understand, as you don't need to open the fixtures file.

@rarkins WDYT?